src/main/resources/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction/index.jelly

```diff
@@ -34,7 +34,7 @@
                     </j:if>
                     <a href="../pipeline-graph" class="jenkins-button">
                         <l:icon src="symbol-file-tray-stacked-outline plugin-ionicons-api"/>
-                        ${%Details}
+                        ${%Graph}
```

Comment:
That's a good point but worry that Overview is a bit vague. I think of it as the Graph view, but that might just be because the link in the side menu says `Pipleiine Graph` - I dropped `Pipeline` because the link is in the Pipeline Console.
<img width="204" alt="Screenshot 2023-03-10 at 21 17 13" src="https://user-images.githubusercontent.com/4447764/224430029-9b1c8a6b-9f63-40c6-a20f-5ffd2e0f58e2.png">

Reply:
Also, I never looked at the icon much before, but seems like it might be an in-tray. That said I it's hard to find a good alternative in ionicon, the charts are all bar charts. The only one I though of was: `<ion-icon name="analytics-outline"></ion-icon>`

Reply:
I think that's just because of the name on the side panel.

yeah that's probably a better icon, the current one is a bit weird

Reply:
WDYT of the `git-network-outline` one?

<img width="659" alt="image" src="https://user-images.githubusercontent.com/91831478/224438945-7bfc30a0-5941-49cc-be57-de686b4dda73.png">